src/bin/cql-stress-cassandra-stress/java_generate/values/mod.rs

```diff
@@ -88,3 +90,16 @@ pub trait ValueGenerator {
         size_distribution: &mut dyn Distribution,
     ) -> Self::Value;
 }
+
+#[cfg(test)]
+mod tests {
+    use super::{blob::Blob, Generator, GeneratorConfig};
+
+    #[test]
+    fn generator_config_salt_test() {
+        let blob_gen = Blob::default();
+        let config = GeneratorConfig::new(String::from("randomstrC0"), None, None);
+        let gen = Generator::new(blob_gen, config);
+        assert_eq!(gen.salt, 5919258029671157411);
```

Comment:
Maybe this is too verbose (you don't have to agree with this suggestion), but maybe there should be a comment that those hardcoded values are taken from running the corresponding code in Java.